android/library/src/main/java/com/liferay/mobile/screens/viewsets/defaultviews/ddl/form/fields/BaseDDLFieldTextView.java

```diff
@@ -40,7 +40,8 @@
     implements DDLFieldViewModel<T>, TextWatcher {
 
     protected TextView labelTextView;
-    protected EditText textEditText;
+	protected TextView hintTextView;
```

Comment:
please be aware of indentation, run detekt and the end of the PR, thanks again :)